### PR TITLE
Add IsChecked property for menu item

### PR DESCRIPTION
**Pull request by ddeltasolutions**

### CODE REVIEW

**Comment by Roemer** [comment]
Could you maybe add a test for this?

File: src/FlaUI.Core.UITests/Elements/MenuTests.cs [hunk 1]
```diff
@@ -53,5 +64,25 @@ public void TestMenuWithSubMenusByName()
             Assert.That(fancy, Is.Not.Null);
             Assert.That(fancy.Properties.Name.Value, Is.EqualTo("Fancy"));
         }
+        
+        [Test]
+        public void TestCheckedMenuItem()
+        {
+            if (ApplicationType == TestApplicationType.WinForms)
+            {
+                return;
```
Comment (Roemer) [inline src/FlaUI.Core.UITests/Elements/MenuTests.cs]: You should use `Assert.Ignore` like in `TextBoxTests.TextBoxColorTest`

File: src/FlaUI.Core.UITests/Elements/MenuTests.cs [hunk 2]
```diff
@@ -29,9 +29,20 @@ public void TestMenuWithSubMenus()
             Assert.That(subitems1, Has.Length.EqualTo(1));
             Assert.That(subitems1[0].Properties.Name, Is.EqualTo("Exit"));
             var subitems2 = items[1].Items;
-            Assert.That(subitems2, Has.Length.EqualTo(2));
+            if (ApplicationType == TestApplicationType.WinForms)
+            {
+                Assert.That(subitems2, Has.Length.EqualTo(2));
```
Comment (Roemer) [inline src/FlaUI.Core.UITests/Elements/MenuTests.cs]: Can you maybe add some comments why this is different?

File: src/TestApplications/WpfApplication/MainWindow.xaml.cs [hunk 3]
```diff
@@ -26,5 +26,20 @@ private void Selector_OnSelectionChanged(object sender, SelectionChangedEventArg
                 }
             }
         }
+        
+        private void OnShowLabel(object sender, RoutedEventArgs e)
```
Comment (Roemer) [inline src/TestApplications/WpfApplication/MainWindow.xaml.cs]: Not really MVVM but for the sake of a demo application, this is fair enough ;)

File: src/FlaUI.Core.UITests/Elements/MenuTests.cs (hunk 1 above)
Comment (ddeltasolutions) [reply src/FlaUI.Core.UITests/Elements/MenuTests.cs]: Done.

File: src/FlaUI.Core.UITests/Elements/MenuTests.cs (hunk 2 above)
Comment (ddeltasolutions) [reply src/FlaUI.Core.UITests/Elements/MenuTests.cs]: Done.

**Comment by Roemer** [comment]
Thanks for your contribution! It is merged now.

**Pull request merged**